client/app/competitor/page.tsx

```diff
@@ -1,3 +1,162 @@
+                </div>
+            </div>
+        </div>
+    );
+};
+
+// TODO: need to be able to grab info from the text editor and have this func take it as a param
+const runTest = () => {
+    return (
+        <div className="mx-4 flex w-full flex-col items-center gap-2">
+            <Button
+                variant="outline"
+                className="h-12 w-full max-w-72 text-white hover:bg-opacity-90"
+            >
+                <b>Test</b>
+            </Button>
+            <Button
+                variant="outline"
+                className="h-12 w-full max-w-72 text-white hover:bg-opacity-90"
+            >
+                <b>Submit</b>
+            </Button>
+        </div>
+    );
+};
+
+const testResults = () => {
+    return (
+        <div className="w-full">
+            <Accordion type="single" collapsible>
+                <AccordionItem value="item-1">
+                    <AccordionTrigger className="items-center justify-between px-8">
+                        <h1>
+                            <b>Test Case 1</b>
+                        </h1>
+                        <h1 className="flex items-center justify-center text-green-500">
+                            <b>PASS</b>
+                        </h1>
+                    </AccordionTrigger>
+                    <AccordionContent className="px-8">
+                        <Code>0 2 11 15</Code>
+                    </AccordionContent>
+                </AccordionItem>
+
+                <AccordionItem value="item-2">
+                    <AccordionTrigger className="items-center justify-between px-8">
+                        <h1>
+                            <b>Test Case 2</b>
+                        </h1>
+                        <h1 className="flex items-center justify-center text-red-700">
+                            <b>FAIL</b>
+                        </h1>
+                    </AccordionTrigger>
+                    <AccordionContent className="px-8">
+                        <Code>2 11 15 0</Code>
+                    </AccordionContent>
+                </AccordionItem>
+
+                <AccordionItem value="item-3">
+                    <AccordionTrigger className="items-center justify-between px-8">
+                        <h1>
+                            <b>Test Case 3</b>
+                        </h1>
+                        <h1 className="flex items-center justify-center text-red-700">
+                            <b>FAIL</b>
+                        </h1>
+                    </AccordionTrigger>
+                    <AccordionContent className="px-8">
+                        <Code>2 15 15 0</Code>
+                    </AccordionContent>
+                </AccordionItem>
+            </Accordion>
+        </div>
+    );
+};
+
 export default function Competitor() {
-    return <h1>Competitor Panel</h1>;
+    return (
+        <div className="h-full">
+            <CompetitorNavbar />
+
+            <div className="flex h-[95vh] w-full">
```

Comment:
this `h-[95vh]` feels like a hack to me... This should probably have a `flex-grow` and have the outer container use flexbox.